package.json

```diff
@@ -6,6 +6,7 @@
     "android:build:dev": "eas build -p android --profile development --local",
     "android:reverse": "adb reverse tcp:8081 tcp:8081 && adb reverse tcp:9875 tcp:9875",
     "android:sendApk": "./scripts/android/sendApk.sh",
+    "android:build:clean": "rm -rf ./android && npx expo prebuild --platform android && yarn android",
```

Comment:
We could add this to the clean script and use the same AppleScript magic to ensure this is immediate!